### PR TITLE
feat: migrate runtime caches recipe

### PR DESCRIPTION
**Pull request by toph-allen**
This PR migrates the runtime caches recipe.

Addresses #54 

Notes, which may require the creation of future issues
- #54 called for adding information about rebuilding content caches. However, that wasn't in the old recipe, and has not been added in this new recipe. We should add a new issue to track that.
- Neither of the SDK packages implement runtime cache deletion with task polling.
  - The original recipe includes a longer, more complex snippet to poll in a loop. I didn't add that, just a single API call to manually check. May be worthwhile preserving the old recipe's approach, but I was unsure how to present it in the format of the new notebook. The SDKs should add support for an interactive cache deletion operation that polls the tasks API to report success or failure.
  - `rsconnect-python` includes such a feature already, but I chose not to use it here.

### CODE REVIEW

File: 10-server-management/10-viewing-runtime-caches/index.qmd [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+## Python
+
+Call Connect's API using the SDK. The API returns a JSON response that can be easily transformed into a DataFrame.
+
+```{.python}
+from posit import connect
+import polars as pl
+
+ABC = "154bd2af-e8fa-4aa4-aab8-dcef701f4af9"
+XYZ = "23"
```
Comment (tdstein) [inline 10-server-management/10-viewing-runtime-caches/index.qmd:22]:
```suggestion
```

File: 10-server-management/10-viewing-runtime-caches/index.qmd [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+client = connect.Client()
+
+response = client.get("/v1/system/caches/runtime")
+caches_df = pl.DataFrame(response.json()["caches"])
+```
+
+The data includes information on the language and version of the cache. For Connect servers using [off-host execution](https://docs.posit.co/connect/admin/getting-started/off-host-install/), the `image_name` column includes the name of the execution image. The name `Local` indicates that the cache is for local execution mode.
+
+```{.python}
+>>> print(caches_df)
```
Comment (tdstein) [inline 10-server-management/10-viewing-runtime-caches/index.qmd:33]: I've been doing this:

```suggestion
```{.python}
>>> import polars as pl
>>> pl.DataFrame(caches_df)
```

File: 10-server-management/10-viewing-runtime-caches/index.qmd [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+## R
+
+Call Connect's runtime caches API using `connectapi` and transform the response into a data frame.
+
+```{.r}
+library(connectapi)
+library(purrr)
+
```
Comment (tdstein) [inline 10-server-management/10-viewing-runtime-caches/index.qmd:61]:
```suggestion
```

File: 10-server-management/11-deleting-runtime-caches/index.qmd [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+instructions = {
+    "language": LANGUAGE,
+    "version": VERSION,
+    "image_name": IMAGE_NAME
+}
+
+response = client.delete(
+    "v1/system/caches/runtime",
+    json = instructions,
```
Comment (tdstein) [inline 10-server-management/11-deleting-runtime-caches/index.qmd:36]:
```suggestion
    json=instructions,
```

It's a Python convention...

File: 10-server-management/11-deleting-runtime-caches/index.qmd [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+```{.python}
+>>> task_status.json()
+{'id': 'lFvijYN16kjCoal6',
+ 'output': ['Deleting runtime cache...', 'Successfully deleted runtime cache'],
+ 'result': None,
+ 'finished': True,
+ 'code': 0,
+ 'error': '',
+ 'last': 2}
+```
```
Comment (tdstein) [inline 10-server-management/11-deleting-runtime-caches/index.qmd:117]: Theres a simple way to do this in the SDK ,but this is fine for now. Also, we can split this information out into separate page and link to it.

**Review by tdstein — APPROVED** [review]
Some minor stuff to change. Looks great overall!

File: 10-server-management/10-viewing-runtime-caches/index.qmd [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+:::{.panel-tabset group="language"}
+
+## Python
+
+Call Connect's API using the SDK. The API returns a JSON response that can be easily transformed into a DataFrame.
+
+```{.python}
+from posit import connect
+import polars as pl
```
Comment (toph-allen) [inline 10-server-management/10-viewing-runtime-caches/index.qmd:19]:
```suggestion
```

File: 10-server-management/11-deleting-runtime-caches/index.qmd (hunk 4 above)
Comment (toph-allen) [reply 10-server-management/11-deleting-runtime-caches/index.qmd:36]: Oh duh yeah. I speak fluent Python most of the time, but code-switching in a single file…

(Side note, it's the only Python convention I don't like, given than equals signs are surrounded by spaces outside of argument lists)

File: 10-server-management/11-deleting-runtime-caches/index.qmd (hunk 5 above)
Comment (toph-allen) [reply 10-server-management/11-deleting-runtime-caches/index.qmd:117]: Yeah, that way we could have slightly more complex code that actually polls and prints updates as status changes.

**Pull request merged**